eras/conway/impl/testlib/Test/Cardano/Ledger/Conway/Imp/LedgerSpec.hs

```diff
@@ -212,6 +216,11 @@ spec = do
 
   describe "Mempool" $ do
     let
+      submitFailingMempoolTx ::
```

Comment:
these type signatures are needed to make the compiler infer correctly the `injectFailure` below